.travis.yml

```diff
@@ -24,6 +24,8 @@ jobs:
         - python3 codalab_service.py start --services default monitor --version ${TRAVIS_BRANCH}
       script:
         - python3 codalab_service.py test default --version ${TRAVIS_BRANCH}
+          # Make sure restarting worker doesn't cause any issues (ie in serialization/deserialization)
+        - docker restart codalab_worker_1 && python3 codalab_service.py test run --version ${TRAVIS_BRANCH}
```

Comment:
Why only run the `run` tests here?

Reply:
I don't think we'd get any extra signal from the other tests at this point since we don't have any dedicated tests for things like dependency/docker image caches or worker state resumption. This is just a sanity check that all the worker sub-components have initialized successfully that they can complete basic runs.